linker/shared/src/main/scala/org/scalajs/linker/backend/emitter/CoreJSLib.scala

```diff
@@ -545,7 +545,7 @@ private[emitter] object CoreJSLib {
             paramList((instance :: args): _*), body)
       }
 
-      // toString()java.lang.String
+      // toString()java.lang.String is special (it is always exported).
```

Comment:
It's not because it is always exported. In fact, the IR spec doesn't specify that `j.l.Object.toString()` must be exported (there is an actual export definition in the hand-written IR of `j.l.Object`).

`toString()` is special here because the IR spec says so: see [here](https://lampwww.epfl.ch/~doeraene/sjsir-semantics/#sec-sjsir-apply-runtime-semantics-evaluation), step 8.